lib/lotus/commands/new/app.rb

```diff
@@ -104,7 +104,7 @@ def upcase_app_name
         end
 
         def classified_app_name
-          Utils::String.new(app_name).classify
+          Utils::String.new(app_name).tr('-', '_').classify
```

Comment:
@davydovanton, @AlfonsoUceda is right `#classify` doesn't work properly and needs to be fixed.

```ruby
➜ bundle console
Resolving dependencies...
irb(main):001:0> require 'lotus/utils/string'
=> true
irb(main):002:0> Lotus::Utils::String.new('app-name').classify.to_s
=> "App-name"
```

It returns `"App-name"`, which is clearly an invalid Ruby class name. We expect `"App::Name"` as result.